solr/solr-ref-guide/src/uploading-structured-data-store-data-with-the-data-import-handler.adoc

```diff
@@ -1,1077 +0,0 @@
-= Uploading Structured Data Store Data with the Data Import Handler
```

Comment:
Should this page stay in 9.x as a placeholder, but with a text explaining that DIH is no longer maintained by the project, pointing to the 3rd party package? Or should such references be in a totally new RefGuide page where all deprecated/removed features that are now packages, are listed?

Reply:
@ctargett , your opinion would be very valued on the documentation part. Currently, the patch deletes all DIH references and pages except for history and new mention in significant-changes. I did it to make sure I did not leave any dangling cross-references or files. We can put it back or cross-link somewhere else as you think is best.

Reply:
It's an interesting question, and I don't have an answer. I dislike trying to keep a mostly blank page around, because each version of the Guide should stand by itself and be for the version it describes - 9.0 won't have DIH so the page shouldn't be there. That's a general principle. But, the reality is people will look for it and may be lost when they don't find it.

I don't have an ideal solution generally, but in this case I coincidentally happen to be planning a mass re-org of the entire Ref Guide (SOLR-14444) for 9.0. One of the things I'm considering is to consolidate the content of some pages which will remove some. Other pages I intend to rename. So, I'll need to find an answer to the problem. I'm keeping track of these changes in a spreadsheet, in case we need it for however we choose to handle it. Right now I've added this page to my list of changes (with other 9.0 removals that have happened), so I'm OK if you go forward removing it with this effort.

Reply:
Ok. I'll keep it removed. Maybe in the next version of the Ref Guide, we can feed the section headers into the search as well and DIH could be in the title of one of the sections to find.